source/includes/2016-03-05/_disputes.md

```diff
@@ -1237,6 +1244,160 @@ chargehound.disputes.update("dp_123",
 | body         | string            | required  |The email body, as plain text.|
 | caption      | string            | optional  |A description of the email.|
 
+## Past payments
+
+Showing a history of valid transactions with customer is good evidence that their disputed transaction is also valid. Typically, Chargehound can automatically fetch past payments from your payment processor. <b>Generally, you do not need to set past payment information yourself.</b> 
```

Comment:
Showing a history of valid transactions with a customer can serve as evidence that their disputed transaction was also a valid transaction.